src/sfctl/custom_version.py

```diff
@@ -0,0 +1,16 @@
+import pkg_resources
+from knack.util import CLIError
+
+def display_versions(client):
+    pkg = pkg_resources.get_distribution("sfctl")
+    sfctl_version = pkg.version
+    target_sf_version = None
+    for dependency in pkg.requires():
+        if dependency.key == 'azure-servicefabric':
+            # specs is a list of tuples for each version specified
+            target_sf_version = dependency.specs[0][1]
```

Comment:
Are you certain that dependency.specs[0][1] will always exist?

Reply:
it will be an error if it doesn't regardless

Reply:
Will that error be useful though? I think it would produce an error message about array index out of bounds, but that doesn't provide any context on what it means in terms of a dependency.